autoload/neomake.vim

```diff
@@ -84,29 +84,29 @@ function! neomake#MakeJob(maker) abort
     endif
 endfunction
 
-function! neomake#GetMaker(name, ...) abort
+function! neomake#GetMaker(maker_id, ...) abort
```

Comment:
Yea, it doesn't need to change. The reason I changed it is because the `a:name` is not really a name in some cases. It's easy for someone new to the project (or even heavily involved) to forget that `a:name` isn't just a string and then end up with errors like the one I'm attempting to fix.

Maybe another name besides `a:maker_id` or `name_or_maker`?